src/main/java/dev/j3fftw/worldeditslimefun/slimefun/Items.java

```diff
@@ -0,0 +1,29 @@
+package dev.j3fftw.worldeditslimefun.slimefun;
+
+import dev.j3fftw.worldeditslimefun.WorldEditSlimefun;
+import io.github.thebusybiscuit.slimefun4.api.items.ItemGroup;
+import io.github.thebusybiscuit.slimefun4.api.items.SlimefunItem;
+import io.github.thebusybiscuit.slimefun4.api.items.SlimefunItemStack;
+import io.github.thebusybiscuit.slimefun4.api.recipes.RecipeType;
+import io.github.thebusybiscuit.slimefun4.libraries.dough.items.CustomItemStack;
+import org.bukkit.Material;
+import org.bukkit.NamespacedKey;
+import org.bukkit.inventory.ItemStack;
+
+public class Items {
+    public static void init(WorldEditSlimefun plugin) {
+        final ItemGroup WESF_GROUP = new ItemGroup(new NamespacedKey(plugin, "world_edit_slimefun"), new CustomItemStack(Material.STONE_AXE, "&fWorld Edit Slimefun (Dummy Group)"));
```

Comment:
please format this. its going over the 120 char line

Reply:
Will be in next commit 🫡